### PR TITLE
Leaderboard: hit the API

### PR DESCRIPTION
**Pull request by mariordgez**
In this activity I continue working on the Leaderboard list app, sending and receiving data from/to the Leaderboard API.

### CODE REVIEW

File: src/storage.js [hunk 1]
```diff
@@ -1,30 +1,27 @@
...
-  }
-
-  static getList() {
-    if (localStorage.getItem('ScoreList') == null) {
-      Storage.saveList(new ScoreList());
-    }
-    const scoreList = Object.assign(
-      new ScoreList(),
-      JSON.parse(localStorage.getItem('ScoreList')),
+  // game ID: m0uJU78hgC1wzCRUBfx8
```
Comment (Dipeshtwis) [inline src/storage.js]: - [x] Remove the commented code to make the codebase cleaner.

File: src/index.js [hunk 2]
```diff
@@ -5,32 +5,59 @@ import '@fortawesome/fontawesome-free/js/regular';
...
+  });
 };
 
 const refreshScore = (event) => {
   event.preventDefault();
-
   window.location.reload();
 };
 
+// new game ID: MOPWIcjFvO9yMNxd0RRq
```
Comment (Dipeshtwis) [inline src/index.js]: - [x] Remove the commented code to make the codebase cleaner.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @mariordgez, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Used `async` and `await` with the `fetch` method to send and receive data.
:+1: App runs as expected.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sinansevgi — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi @mariordgez , 

Your code base is cleaner now :clap:
Your project is complete!  
Congratulations! 🎉

You can merge this pull request and continue with your journey :wave: 


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**